ui/lib/pki/addon/components/pki-key-parameters.js

```diff
@@ -23,7 +23,10 @@ const KEY_BITS_OPTIONS = {
 
 export default class PkiKeyParameters extends Component {
   get keyBitOptions() {
-    return KEY_BITS_OPTIONS[this.args.model.keyType];
+    if (Object.keys(KEY_BITS_OPTIONS).includes(this.args.model.keyType)) {
```

Comment:
I think this changes the functionality of the form that I mentioned in my comment above for generating a key

Reply:
I didn't realize this broke the intended functionality, rolling back